R/SVG.R

```diff
@@ -1,14 +1,46 @@
 #' An SVG Graphics Driver
 #'
-#' This function produces graphics compliant to the current w3 svg XML standard.
-#' The driver output is currently NOT specifying a DOCTYPE DTD.
+#' This function produces graphics compliant to the current w3 svg XML
+#' standard. The driver output is currently NOT specifying a DOCTYPE DTD.
+#'
+#' svglite provides two ways of controlling fonts: system fonts
+#' aliases and user fonts aliases. Supplying a font alias has two
+#' effects. First it determines the \code{font-family} property of all
+#' text anchors in the SVG output. Secondly, the font is used to
+#' determine the dimensions of graphical elements and has thus an
+#' influence on the overall aspect of the plots.
+#'
+#' By default, fontconfig is used to find fonts corresponding to the R
+#' families of fonts (\code{sans}, \code{serif}, \code{mono},
+#' \code{symbol}). You can supply the name of an installed font with
+#' the \code{system_fonts} argument. Fontconfig will be used to
```

Comment:
Where does fontconfig come from? It's a bit unclear here - is it an R package, a function, ...?